### PR TITLE
 saved data to db

### PR DESCRIPTION
**Pull request by anthonyltam**
@snrichards hey can you review my code?

### CODE REVIEW

File: database/sample.js [hunk 1]
```diff
@@ -4,11 +4,33 @@ const faker = require('faker');
...
+    }
+   
+    data.push(obj)
+  
+}
 // adds data into db here.
 const generateData = function() {
   // creates new data in database, then disconnects?
-  newHost.create()
+    console.log('blue')
```
Comment (snrichards) [inline database/sample.js]: @anthonyltam This console log should be removed

File: database/index.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
 const mongoose = require('mongoose');
-mongoose.connect('mongodb://localhost/host');
+const mongoUri = 'mongodb://localhost/host';
 
-const db = mongoose.connection;
+// mongoose.connect('mongodb://localhost/host');
```
Comment (snrichards) [inline database/index.js]: @anthonyltam this comment is no longer needed

**Review by snrichards — CHANGES_REQUESTED** [review]

**Comment by anthonyltam** [comment]
@snrichards hey I made those changes, will you take another look?

**Comment by snrichards** [comment]
@anthonyltam looks good!

**Pull request merged**